src/BinaryParsers/PEBinary/ProgramDatabase/Pdb.cs

```diff
@@ -175,7 +178,7 @@ public DisposableEnumerable<SourceFile> CreateSourceFileIterator()
 
         public DisposableEnumerable<SourceFile> CreateSourceFileIterator(Symbol inObjectModule)
         {
-            return new DisposableEnumerable<SourceFile>(this.CreateSourceFileIteratorImpl(inObjectModule.UnderlyingSymbol));
+            return new DisposableEnumerable<SourceFile>(this.CreateSourceFileIteratorImpl(inObjectModule?.UnderlyingSymbol));

```

Comment:
>return new DisposableEnumerable<SourceFile>(this.CreateSourceFileIteratorImpl(inObjectModule?.UnderlyingSymbol)); [](http://example.com/codeflow?start=12&length=113)

This change is now covered by a test `PdbShouldCreateEmptySourceFileIteratorIfInObjectModuleIsNull`